packages/webamp-modern-2/src/skin/AudioPlayer.ts

```diff
@@ -2,6 +2,10 @@ import { clamp, Emitter } from "../utils";
 
 const BANDS = [60, 170, 310, 600, 1000, 3000, 6000, 12000, 14000, 16000];
 
+export const STATUS_PAUSED = -1;
+export const STATUS_STOPPED = 0;
+export const STATUS_PLAYING = 1;
```

Comment:
I think it would be easier to debug if we used strings here and converted them to numbers via a `switch` statement in `SystemObject.getstatus`.

Reply:
done